tests/other_tests.go

```diff
@@ -40,3 +40,13 @@ func TestRunAdvancedExample(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunEnterpriseExample(t *testing.T) {
```

Comment:
This test wont work on GEDev account, and when I tried to add it to DAF enterprise, we hit issues due to clashes with custom roles. I have a [tracking issue](https://github.com/terraform-ibm-modules/terraform-ibm-scc-workload-protection/issues/216) for this, so just remove for now